arm-sql/2015-05-01-preview/swagger/advisors.json

```diff
@@ -0,0 +1,981 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/advisors": {
+      "get": {
+        "tags": [
+          "DatabaseAdvisors"
+        ],
+        "description": "Returns a list of Database Advisors.",
```

Comment:
It matters because this help text will be visible in https://docs.microsoft.com and Azure CLI command descriptions